DER/DerivativesContracts/CommoditiesContracts.rdf

```diff
@@ -73,11 +73,12 @@
 		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISO 10962, Securities and related financial instruments - Classification of financial instruments (CFI) code, Fourth Edition, October 2019</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-drc-comm;BaseIndustrialMetal">
+	<owl:Class rdf:about="&fibo-der-drc-comm;BaseMetal">
 		<rdfs:subClassOf rdf:resource="&fibo-der-drc-comm;Metal"/>
-		<rdfs:label xml:lang="en">base industrial metal</rdfs:label>
+		<rdfs:label xml:lang="en">base metal</rdfs:label>
 		<owl:disjointWith rdf:resource="&fibo-fnd-pas-pas;PreciousMetal"/>
-		<skos:definition xml:lang="en">metal or alloy, other than a precious metal or noble metal, such as copper, lead, zinc, tin, or brass</skos:definition>
+		<skos:definition xml:lang="en">common metal that tarnishes, oxidizes, or corrodes relatively quickly when exposed to air or moisture, that is widely used in commercial and industrial applications, such as construction and manufacturing</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Base metals or alloys include metals other than precious metals, such as copper, lead, zinc, tin, or brass. Note that iron and steel are included under metal and metal products in some classification schemes - see https://fred.stlouisfed.org/series/WPU101 for example.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
IMO excluding iron and steel would be the exception.
Begs the question as to whether we should explicitly represent the different Schemes as such. But see my email about Corporate Actions use of classifier.